pkg/workloads/spark_job/spark_util.py

```diff
@@ -220,53 +229,61 @@ def ingest(ctx, spark):
         df = read_parquet(ctx, spark)
 
     input_type_map = {f.name: f.dataType for f in df.schema}
-
-    for raw_column_name in ctx.raw_columns.keys():
+    inferred_col_type_map = {c.name: c.dataType for c in df.schema}
```

Comment:
`input_type_map` is the same as `inferred_col_type_map`. Probably `input_type_map` makes sense, since for parquet we don't infer it